routing/routes.go

```diff
@@ -373,7 +373,7 @@ func NewRoutes() (r *Routes) {
 			handlers.ServiceDetails,
 			true,
 		},
-		// swagger:route GET /namespaces/{namespace}/apps/{app}/spans traces spansList
+		// swagger:route GET /namespaces/{namespace}/apps/{app}/spans traces appSpans
 		// ---
 		// Endpoint to get Jaeger spans for a given service
```

Comment:
I guess this comment would need to be updated.

I see that for spans we have "AppSpans", I wonder if now that there are a "AppTraces" / "ServiceTraces" if that would require a "AppSpans" / "ServiceSpans".

Perhaps not, and "AppSpans" work for both scenarios, but just leaving a note here.

Reply:
I'm updating the comment, it works for apps indeed, not services.

That's right that we should as well create endpoints for service spans and workload spans. I'll create a follow-up task.

Reply:
Ok, so then in this scenario:
![image](https://user-images.githubusercontent.com/1662329/92230639-c333ed80-eeab-11ea-803e-7b11e173d696.png)
Spans are fetched by app but it should filter by service, too.

Reply:
https://github.com/kiali/kiali/issues/3175